packages/design-system-docs/src/pages/components/Dropdown/Dropdown.example.jsx

```diff
@@ -36,6 +36,13 @@ ReactDOM.render(
       label="Medium dropdown example"
       name="medium_dropdown_field"
     />
+    <Dropdown
+      options={dropdownOptions}
+      defaultValue={'1'}
+      errorMessage="Error message example"
+      label="Dropdown example"
```

Comment:
This looks good, one change that would be helpful would be to update the label to be `Error dropdown example` to be consistent with the naming convention

Reply:
Done. Thanks for picking that up.